imports/api/blocks/server/methods.js

```diff
@@ -463,10 +465,11 @@ Meteor.methods({
                     if (lastSyncedTime){
                         let dateLatest = new Date(blockData.time);
                         let dateLast = new Date(lastSyncedTime);
-                        let genesisTime = new Date(genesisTime);
+
+                        // calculating time to generate average block time 
+                        let genesisTimeStamp = new Date(genesisTime);
                         timeDiff = Math.abs(dateLatest.getTime() - dateLast.getTime());
```

Comment:
It doesn't look like you are using timeDiff in the calculations, are the calculations correct?

Reply:
all the calculations are done in flat and on fronted these are displayed in 2 decimal